example/redash_trino_cluster/main.tf

```diff
@@ -0,0 +1,277 @@
+locals {
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  trino_datastore = jsonencode({
+    catalog : "hive",
+    host : "127.0.0.1",
+    port : 8080,
+    schema : "default",
+    username : "trino"
+  })
+}
+
+module "redash" {
+  source = "../.."
+
+  # redash
+  service         = "redash"
+  host            = "127.0.0.1"
+  port            = 5000
+  container_image = "gitlab-container-registry.minerva.loc/datainn/redash-rabbit-edition:latest"
+  use_canary      = false
+  redash_config_properties = ["python /app/manage.py database create_tables",
+    "python /app/manage.py users create_root admin@mail.com admin123 --password admin --org default",
+    format("python /app/manage.py ds new \\\"trino\\\" --type \\\"trino\\\" --options '%s' --org default", trimsuffix(trimprefix(jsonencode(local.trino_datastore), "\""), "\"")),
```

Comment:
Ooof, kan se for meg det har gått med litt tid til dette 😅

Reply:
Dette er ikke optimalt, men jeg måtte gi meg :sweat:

Reply:
Hehe, det var ikke ment negativt. Kunne kanskje vært en ide å bake dette inn som shell scripts i redash-rabbit-edition imaget?

Reply:
Egentlig kan man definere multi-line string på denne måten i terraform:
```hcl
string_example = <<EOF
This is a long
long
long
string
EOF
```
Dersom jeg bruker denne syntaksen så slipper jeg å håndtere double quotes på denne måten `///"`. Men av en eller annen grunn fikk jeg ikke denne syntaxen til å fungere i tf-fila... 


Jeg er usikker på om det er en god ide å legge dette inn i image, fordi jeg vet ikke hvordan vi skal løse dette når vi integrere vault. Vi kan ta en prat om det.

Reply:
Trenger vi vault nå i første omgang? Trino er jo åpent og trenger ikke auth 🤔

Reply:
Nei, vi trenger ikke vault i dette eksempelet (så sant username alltid er `trino`). Men med en gang en bruker av modulen ønsker å benytte for eksempel postgres som data source (og brukernavn/passord er i Vault), så er ikke denne løsningen tilfredstillende.

Reply:
Hm, ja jeg ser den. Trino slik den er satt opp har ikke noe sjekk på hva brukernavn er, bare at den blir sendt med som `X-Trino-User` i http requesten. Kanskje vi ikke burde implementere en så generisk datasource løsning i første omgang om det fører til ytterligere komplikasjoner som vault osv?